DirectEmployers/MyJobs
# [PD 3154] Enforce readonly non custom CBO's

## What's this PR do?
PR enforces readonly for custom CBO's

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the CBO application

## How should this be manually tested?
Pull down the repository. Go inside of the CBO application. When you go to ANY CBO now the rules should be editable regardless of whether or not it's custom or not.

## Any background context you want to provide?
Second time for this PR being up. The only change is that no matter whether the CBO is custom or not the rules will always be editable per Hal.
